cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -191,6 +218,21 @@ subroutine init_dyn (dt)
             fcor_blk(i,j,iblk) = c2*omega*sin(ULAT(i,j,iblk)) ! 1/s
          endif
 
+         if (grid_system == 'CD') then
+
+         if (trim(coriolis) == 'constant') then
+            fcorE_blk(i,j,iblk) = 1.46e-4_dbl_kind ! Hibler 1979, N. Hem; 1/s
+            fcorN_blk(i,j,iblk) = 1.46e-4_dbl_kind ! Hibler 1979, N. Hem; 1/s
+         else if (trim(coriolis) == 'zero') then
+            fcorE_blk(i,j,iblk) = 0.0
+            fcorN_blk(i,j,iblk) = 0.0
```

Comment:
these should be `c0`, no ? (and the same for the existing code)

Reply:
Good catch. This might change answers for zero coriolis tests.